tensorflow/core/util/ctc/ctc_beam_search_test_float.cc

```diff
@@ -25,8 +25,8 @@ limitations under the License.
 namespace {
 
 typedef std::vector<std::vector<std::vector<float>>> TestData;
-using tensorflow::ctc::CTCBeamSearchDecoder;
-using tensorflow::ctc::CTCDecoder;
+typedef tensorflow::ctc::CTCBeamSearchDecoder<float> CTCBeamSearchDecoder;
+using tensorflow::ctc::CTCDecoder<double> CTCDecoder;
```

Comment:
this is test_float.cc but you're using doubles.  again, please make these templated on T and just use macros to build test functions for both float and double.

Reply:
resolved in 51cd5877412f9d67f2ec180c8087ae8d7a059da1